data/ecosystems/s/skale-network.toml

```diff
@@ -1,17 +1,78 @@
 # Ecosystem Level Information
 title = "Skale Network"
 
-sub_ecosystems = ["Owl Protocol", "Razor Network"]
+sub_ecosystems = [
+    "3Air",
+    "Bit Hotel",
+    "BlockDaemon",
+    "Blockscout",
+    "BlockWallet",
+    "Clet Name Service",
+    "CookBook",
+    "Covalent",
+    "CryptoBlades",
+    "Crypto Colosseum",
+    "Curio",
+    "DappRadar",
+    "Dirac Finance",
+    "Dirt Road Development",
+    "Dune Analytics",
+    "Enkrypt",
+    "Europa Hub",
+    "Exorde",
+    "Fireside",
+    "Hatchy Pocket",
+    "Human Protocol",
+    "Meson",
+    "MyEtherWallet",
+    "Lattice",
+    "l2beat",
+    "NFTb",
+    "NFTrade",
+    "Owl Protocol",
+    "Particle Network",
+    "Pixudi",
+    "Portis",
+    "Razor Network",
+    "ReHold",
+    "Ruby Exchange",
+    "Score Detect",
+    "Snapshot",
+    "StrayShot",
+    "Subsquid",
+    "Tank Wars Zone",
+    "The Graph",
+    "Wagmi",
+    "WalletConnect",
+    "Web3 Auth",
+    "XDAO",
+    "XP Network"
+]
 
 github_organizations = ["https://github.com/skalenetwork"]
 
 # Repositories
 [[repo]]
 url = "https://github.com/Abbas-Khann/NFTicket"
 
+[[repo]]
+url = "https://github.com/ethereum/remix-project"
```

Comment:
This belongs in Ethereum.

Reply:
Hey @jubos 

I can remove this, however, the reason I chose to add it is because I PRd on behalf of SKALE Network to have SKALE Chains added directly to the Remix dropdown. It's also a heavily used tool by SKALE Developers.

Reply:
Remoed in a720d5e0